src/main/java/edu/matc/covidPulse/servlet/Home.java

```diff
@@ -0,0 +1,97 @@
+package edu.matc.covidPulse.servlet;
```

Comment:
As per our convo on slack, I think this is overkill. It looks great, but we don't need a front end (other than documentation / user guide). I think it will be safe to keep all of that contained in the index.jsp, without the need for a servlet.

Reply:
I'll work on it.